tests/tests.py

```diff
@@ -99,6 +98,21 @@ def test_different_backend(self):
         self.assertExceptionMessageEquals(exc.exception,
                                           "Can only render blocks from the Django template backend.")
 
+    @modify_settings(
+        INSTALLED_APPS={
+            'prepend': [
+                'django.contrib.auth',
+                'django.contrib.contenttypes',
```

Comment:
Any particular reason these apps were added? Auth seems kind of weird since nothing about auth is configured!

Reply:
Without these two apps the tests are providing following warning messages:

```python
python manage.py test
Creating test database for alias 'default'...
......../Users/krykowski/workspace/django-render-block/.venv/lib/python3.7/site-packages/django/contrib/contenttypes/models.py:161: RemovedInDjango19Warning: Model class django.contrib.contenttypes.models.ContentType doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class ContentType(models.Model):

/Users/krykowski/workspace/django-render-block/.venv/lib/python3.7/site-packages/django/contrib/auth/models.py:41: RemovedInDjango19Warning: Model class django.contrib.auth.models.Permission doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class Permission(models.Model):

/Users/krykowski/workspace/django-render-block/.venv/lib/python3.7/site-packages/django/contrib/auth/models.py:98: RemovedInDjango19Warning: Model class django.contrib.auth.models.Group doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class Group(models.Model):

/Users/krykowski/workspace/django-render-block/.venv/lib/python3.7/site-packages/django/contrib/auth/models.py:436: RemovedInDjango19Warning: Model class django.contrib.auth.models.User doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class User(AbstractUser):

.......ss...ss
----------------------------------------------------------------------
Ran 22 tests in 0.043s

OK (skipped=4)
Destroying test database for alias 'default'...
```

That's related to the fact we are using the `RequestFactory` and each request is operating on user entry (anonymous or authenticated). We can get rid of it, by replacing the request with mock implementation like:

`request = Mock(path=u'/dummy-url')`

Thoughts? I didn't want to introduce mock in here, as it's more important to test `render_block_to_string` with real request instance.

Reply:
Ah! I didn't realize that `RequestFactor` depended on `auth` being installed. This is fine then!